source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -3,278 +3,528 @@
-   run::
+   Note that mixer automatically looks for a file named 
+   :file:`release-image-config.json`, but you can choose whatever name you want.
+   To use a different name, simply pass the 
+   :option:`--template path/to/file.config` flag when creating your image.
 
-      # sudo mixer build-image -format 1
+   Edit the config file to include all bundles that you want *preinstalled*
+   into your image. The rest of the bundles in your mix will be available to your users via:
 
-   The output from this should be an image that is bootable as a VM or
-   installable to baremetal.
+   .. code-block:: bash 
 
-   .. note::
-      You need to pass in -format <FORMAT_NUMBER> if the format you are
-      building is different than the format of Clear Linux OS you are currently
-      building on. Format version can be found via
+      # swupd bundle add 
 
-   .. code-block:: console
+   Keeping this list small allows for a smaller image size. For a minimal,
+   base image, this list would be:
 
-      # cat /usr/share/defaults/swupd/format
+      .. code-block:: console
 
-Creating your next Mix version
-------------------------------
+      "Bundles": ["os-core", "os-core-update", "kernel-native"]
+
+   Next, set the ``Version`` field to the mix version content that the image
+   should be built from. ``ister`` allows you to build an image from any mix
+   version that you have built, not just the current one. For the first build
+   example we've been using, ``Version`` would be set to 10.
+
+   Finally, to build the image, run:
 
-#. **Initialize next Mix version info**. To update the versions and prep for
-   your next mix:
+   .. code-block:: bash
 
-   Update the .mixversion file to the next version number you want to build.
-   From this point you can iterate through, starting again at step 4 and doing
-   modifications as needed. For example:
+      # sudo mixer build image --format 1
 
-   - Add/Remove/Modify Bundles
-   - sudo mixer build-chroots
-   - sudo mixer build-update
-   - (Optionally) sudo mixer-pack-maker.sh --to <NEWVERSION> --from <PREV_VERSION> -S /home/clr/mix/update
+   The output from this should be an image that is bootable as a virtual
+   machine and can be installed on bare metal.
+
+   .. note::
+      By default, ``ister`` uses the format version of the build machine it
+      is running on. Therefore, if the format you are building is different
+      than the format of the Clear Linux OS that you are building on, you
+      need to pass :option:`--format <FORMAT_NUMBER>`. You can find your
+      current format version by running:
 
+      .. code-block:: bash
 
-#. **Update Bundles (Optional)**.  Update ``clr-bundles``.  In the workspace,
-   run::
+         # cat /usr/share/defaults/swupd/format
 
-    # sudo mixer get-bundles
+Creating your next mix version
+==============================
+
+**Update the next Mix version info**. Update the :file:`.mixversion` file to
```

Comment:
@kevincwells - .mixversion should be just mixversion.

Reply:
Oh, I knew I was forgetting one of the many new commands. This section needs to be updated to instead use the new `mixer versions update` command.